lib/methods/import.js

```diff
@@ -743,12 +766,21 @@ module.exports = self => {
             }
           }
 
+          const replaceItem = replaceAposDocIds.find(([ importId ]) => importId === doc.aposDocId);
           await self.insertOrUpdateDoc(req, {
             doc,
             method: 'update',
-            failedIds
+            failedIds,
+            existingAposDocId: replaceItem?.[1]
```

Comment:
I think we should send an object, it's not very clear to understand what is inside this array.

Reply:
See the comment above about the same in the UI. We are very consistent about it and it follows what the core does. Let me know if you still want it an object and I'll do it.

Reply:
Ok in that case maybe you can just destructure the array to store each element in a variable, it would make it more readable / comprehensible I think:
```javascript
          const [ aposDocId, replaceId ] = replaceAposDocIds
            .find(([ importId ]) => importId === doc.aposDocId) ?? [];
          await self.insertOrUpdateDoc(req, {
            doc,
            method: 'update',
            failedIds,
            existingAposDocId: replaceId
          });

          if (replaceId) {
            replaceDocIdPairs.push([
              doc._id,
              `${replaceId}:${doc.aposLocale}`
            ]);
          }
```